gcp/modules/couchdb/recover.tf

```diff
@@ -0,0 +1,83 @@
+# This resource is needed to restore PVs that were previously claimed by CouchDB
```

Comment:
Is it possible to somehow also use this code for (manual) data recovery scenarios? Seems like it does a lot of the manual steps from the README.

Maybe not worth it at this time?

Reply:
It does not do more things from README than it actually does :)
It does not verify snapshots, delete disks, create new disks from snapshots, etc.

We can make it do all these things, but yeah, probably not worth it this time...